### PR TITLE
Fixing bugs in docs for MarketBidCost

### PR DESCRIPTION
**Pull request by sourabhdalvi**

### CODE REVIEW

File: docs/src/modeler_guide/market_bid_cost.md [hunk 1]
```diff
@@ -53,16 +53,16 @@ pairs (cost in $/p.u-hr & power-point in p.u-hr), which is modeled same as TwoPa
 ThreePartCost. Code below shows an example of how to build a TimeSeriesData.
 
 ```@example
-Using PowerSystems
+using PowerSystems
 data =
-    Dict(Dates.DateTime("2020-01-01") => [
+    Dict(PowerSystems.Dates.DateTime("2020-01-01") => [
```
Comment (jd-lara) [inline docs/src/modeler_guide/market_bid_cost.md:58]: Why this? just add using Dates.

File: docs/src/modeler_guide/market_bid_cost.md [hunk 2]
```diff
@@ -53,16 +53,16 @@ pairs (cost in $/p.u-hr & power-point in p.u-hr), which is modeled same as TwoPa
 ThreePartCost. Code below shows an example of how to build a TimeSeriesData.
 
 ```@example
-Using PowerSystems
+using PowerSystems
```
Comment (jd-lara) [inline docs/src/modeler_guide/market_bid_cost.md:56]:
```suggestion
using PowerSystems, Dates
```

File: docs/src/modeler_guide/market_bid_cost.md [hunk 3]
```diff
@@ -53,16 +53,16 @@ pairs (cost in $/p.u-hr & power-point in p.u-hr), which is modeled same as TwoPa
...
+    Dict(PowerSystems.Dates.DateTime("2020-01-01") => [
         [(0.0, 0.05), (290.1, 0.0733), (582.72, 0.0967), (894.1, 0.120)],
-        [(0.0, 0.05), (300.1, 0.0733), (600.72, 0.0967), (900.1, 0.120)]
+        [(0.0, 0.05), (300.1, 0.0733), (600.72, 0.0967), (900.1, 0.120)],]
     )
 time_series_data = Deterministic(
     name = "variable_cost",
     data = data,
-    resolution = Dates.Hour(1)
+    resolution = PowerSystems.Dates.Hour(1)
```
Comment (jd-lara) [inline docs/src/modeler_guide/market_bid_cost.md:65]: Let's not do this.

**Review by jd-lara — CHANGES_REQUESTED** [review]
Import dates like we usually do, the syntax for dates is not a good one

File: docs/src/modeler_guide/market_bid_cost.md (hunk 1 above)
Comment (sourabhdalvi) [reply docs/src/modeler_guide/market_bid_cost.md:58]: Dates isn't in the docs environment

**Pull request merged**